apitools/gen/gen_client_lib.py

```diff
@@ -240,10 +241,11 @@ def WriteSetupPy(self, out):
         printer('import setuptools')
         printer('REQUIREMENTS = [')
         with printer.Indent(indent='    '):
-            if self.apitools_version.startswith('0.4.'):
-                printer('"google-apitools>=0.4.8,<0.5",')
-            else:
-                printer('"google-apitools==%s",', self.apitools_version)
+            match = re.search(
```

Comment:
the re works here, but i'd rather do something like

  parts = self.apitools_version.split('.')
  major = parts.pop(0)
  minor = parts.pop(0)

and use `major` and `minor` below. it's a little simpler, but it'll work even if we release a version 0.6 or 1.0 one day (which would fail to match the re).